bundle/edu.gemini.phase2.skeleton.servlet/src/main/scala/edu/gemini/phase2/skeleton/factory/package.scala

```diff
@@ -1,4 +1,38 @@
 package edu.gemini.phase2.skeleton
 
+import edu.gemini.model.p1.immutable.{Band, Observation, Proposal, TimeAmount}
+
+import scalaz._
+import Scalaz._
+
 package object factory {
+  // Filter to only the enabled observations by appropriate band.
+  def enabledObs(proposal: Proposal): List[Observation] = {
+    val band = extractBand(proposal)
+    proposal.observations.filter(obs => obs.band == band && obs.enabled)
```

Comment:
We should double check that what's exported from ITAC as enabled corresponds here